db/migrate/20170619161514_add_options_to_ext_management_system.rb

```diff
@@ -0,0 +1,34 @@
+class AddOptionsToExtManagementSystem < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    serialize :options
+  end
+  class CustomAttribute < ActiveRecord::Base
+    serialize :serialized_value
+  end
+
+  def up
+    add_column :ext_management_systems, :options, :text
+
+    say_with_time("Migrating Kubernetes provider options") do
+      ExtManagementSystem.where(:type => ["ManageIQ::Providers::Kubernetes::ContainerManager",
+                                          "ManageIQ::Providers::Openshift::ContainerManager"]).each do |cp|
+        migrated_options = {}
+        CustomAttribute.where(
+          :resource_type => cp.type,
+          :resource_id   => cp.id,
+          :section       => 'cluster_settings',
+          :name          => %w(no_proxy http_proxy https_proxy)
+        ).each do |ca|
+          migrated_options["image_inspector_#{ca.name}".to_sym] = ca.value
+          ca.delete
```

Comment:
@enoodle isn't it risky to delete this before having it persisted in the new options? (`cp.update`)